### ISSUE TITLE
new error in version 2.1.1

### ISSUE THREAD

**Issue by skypce** [jrnewell/spotify-ripper#63]
Spotify error detected
Operation did not complete in 10.000s
Skipping to next track...
Logging out...

**Comment by jrnewell** [comment]
Yeah, I am fairly certain this is related to this upstream issue https://github.com/mopidy/pyspotify/issues/119 which I think is being caused by ``libspotify`` bug (I can reproduce it even in libspotify's example code).  I doubt it will be fixed because libspotify is deprecated and not under active development - sorry.

**Comment by fratellolupo** [comment]
They mention a workaround:
browser = album.browse()
browser.load()

**Comment by jrnewell** [comment]
Yeah, I have that workaround implemented for albums, but I am not sure how to do it for tracks.  I was going to try to test loading a random album before loading this track, but it looks like the track loads properly today.
